### ISSUE TITLE
SURVIVOR merge AVGLEN positive when SVTYPE=DEL

### ISSUE THREAD

**Issue by mpauper** [fritzsedlazeck/SURVIVOR#47]
Hi Fritz,

Despite this:
```
convert << ";AVGLEN=";
	if(entry->type == 0){
		convert << get_avglen(entry->caller_info)*-1;
	}else if (entry->type != 3) {
		convert << get_avglen(entry->caller_info);
	} else {
		convert << "0";   // TODO think about it.
	}
```

I am running into occurences of merged deletions that have a positive AVGLEN.

This is with the latest 1.0.5 version.

**Comment by fritzsedlazeck** [comment]
Thanks I will look into it by Thursday.
Cheers
Fritz

**Comment by mpauper** [comment]
Hi Fritz,

Just realized this is related and only happens in cases where different SV types get merged, like I reported in issue #44

**Comment by fritzsedlazeck** [comment]
Ah that can be. I currently only check if its DEL only for making the length negative. 
I still havent had time to look into it, but i should have time tomorrow.
Thanks
Fritz
